### ISSUE TITLE
Request for dataset

### ISSUE THREAD

**Issue by shahrzadnikbakhtian** [petermlakar/ANET2#1]
Hi,

I am a data science master's student at Oslomet University. Your paper titled "Ensemble weather forecast post-processing with a flexible probabilistic neural network approach" is relevant to my research area. I am working on a normalizing flow algorithm to forecast wind speed, and it would be great if you helped me on this path. I need to reproduce your work by using your GitHub instance "https://github.com/petermlakar/ANET2". At this stage, the problem is that I can not access the datasets that you have used, "ESSD_benchmark_test_data_forecasts.nc" and "ESSD_benchmark_test_data_observations.nc", I tried to retrieve data from the original site, "https://eupp-benchmark.github.io/EUPPBench-doc/files/EUPPBench_datasets.html#stations-data" but I could not figure out how to get the data. Is it possible for you to send the .nc files over?
I appreciate your contribution. 

Best regards,
Shahrzad

**Comment by petermlakar** [comment]
Hello Shahrzad,

while I can not provide you with the data we used during the EUPPBench benchmark I can direct you to the publicly available portion of the data on the [EUPPBench GitHub repository](https://github.com/EUPP-benchmark/ESSD-benchmark-datasets). The data that can be downloaded from there is in the correct format and stored as .nc files.
It does, however, not include the Swiss stations, as they are not freely accessible. Therefore, the results you'll get with the ANET2 method will differ from the ones reported in the paper, since the Swiss stations form almost half of the dataset with stations in complex Alpine terrain.

I have trained the ANET2 Flow model on the aforementioned dataset, with out-of-the-box hyperparameter values, and got the following results on the test dataset, in terms of CRPS:
![CRPS](https://github.com/petermlakar/ANET2/assets/11166814/59395dd6-6eb8-4b67-89b6-7494006a9809)
I suspect that you can get better results by tweaking the model parameters, as I believe the model is over-parameterised in this setting. 

I have updated the repository with a new guide on how to use the framework and how to structure your project to quickly get the training, generation, and evaluation going.

If you have any more questions, do not hesitate to ask!

Kind regards, 
Peter

**Comment by shahrzadnikbakhtian** [comment]
Hi Peter,

Thank you so much for your support! I believe that your explanations will help me a lot. I am starting to train the model with the provided dataset and then my dataset.

Best Regards,
Shahrzad
